TestPurgeFileHoldingLock fails

--- FAIL: TestPurgeFileHoldingLock-4 (0.02s)
	purge_test.go:123: filenames = [5.test 6.test 7.test 8.test 9.test], want [7.test 8.test 9.test]
FAIL
coverage: 80.3% of statements
FAIL	github.com/coreos/etcd/pkg/fileutil	0.147s

--- a/pkg/fileutil/purge_test.go
+++ b/pkg/fileutil/purge_test.go
@@ -86,7 +86,7 @@ func TestPurgeFileHoldingLock(t *testing.T) {
 
 	stop := make(chan struct{})
 	errch := PurgeFile(dir, "test", 3, time.Millisecond, stop)
-	time.Sleep(5 * time.Millisecond)
+	time.Sleep(20 * time.Millisecond)
 
 	fnames, err := ReadDir(dir)
 	if err != nil {
@@ -112,7 +112,7 @@ func TestPurgeFileHoldingLock(t *testing.T) {
 		t.Fatal(err)
 	}
 
-	time.Sleep(5 * time.Millisecond)
+	time.Sleep(20 * time.Millisecond)
 
 	fnames, err = ReadDir(dir)
 	if err != nil {

